### ISSUE TITLE
Stop using red to indicate failure

### ISSUE THREAD

**Issue by blacklistisnotracist** [sebastianbergmann/phpunit#4287]
https://github.com/sebastianbergmann/phpunit/blob/cdb9cbcd79ee45558ae2b16e51943d0950b27ae3/src/Util/TestDox/CliTestDoxPrinter.php#L69

Thank you for your heads-up on the usage of more inclusive languages (#4272, #4278, #4273, #4274). On top of these, could we also stop using red color to indicate failure? I'm Native American and I always find it offensive when I see an error or failure message painted with red.

**Comment by epdenouden** [comment]
Is this real life?

**Comment by blacklistisnotracist** [comment]
@epdenouden Hi, I'm trying not to be offensive here but would you mind telling me why my issue has been closed? Red is always associated with danger, error, failure, etc. and every time I see these text messages painted in red, it reminds me of our horrendous history slaughtered by Western people. If we are removing blacklist/whitelist because of the recent BLM movement, why only red people are treated any differently than black people?

**Comment by blacklistisnotracist** [comment]
@sebastianbergmann

**Comment by epdenouden** [comment]
Go have an offline chat with some actual humans involved with BLM and present the argument above to them. Living in the international hub city of Amsterdam, I have this actual discussion with people close to me. I heartily support the recent changes.

The time you spent [anonymously trolling](https://github.com/blacklistisnotracist/the-most-politically-correct-software) could have been used to bring something to the project, either in code or in _community_. At the end of the day it is _people_ using our highly complex tools.

TL;DR the frogboys already claimed green so we had to pick something ¯\_(ツ)_/¯
